From: Bernhard Walle <bernhard@bwalle.de>
Date: Sat, 21 Jan 2012 22:00:14 +0100
Subject: [PATCH] Darwin: Fix problem with __block parameter name

This fixes the following compile error:

-------------------- 8< ---------------------------
./glibc/malloc/obstack.h:190: error: __block attribute can be specified
on variables only
In file included from glibc/locale/programs/charmap.c:32:
-------------------- >8 ---------------------------

Compiler war gcc-4.2 [i686-apple-darwin11-gcc-4.2.1 (GCC) 4.2.1 (Apple
Inc. build 5666) (dot 3)].

Signed-off-by: Bernhard Walle <bernhard@bwalle.de>
---
 eglibc/malloc/obstack.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/eglibc/malloc/obstack.h b/eglibc/malloc/obstack.h
index 449070e..0c4f2e4 100644
--- a/eglibc/malloc/obstack.h
+++ b/eglibc/malloc/obstack.h
@@ -187,7 +187,7 @@ extern int _obstack_begin_1 (struct obstack *, int, int,
 			     void (*) (void *, void *), void *);
 extern int _obstack_memory_used (struct obstack *);
 
-void obstack_free (struct obstack *__obstack, void *__block);
+void obstack_free (struct obstack *__obstack, void *);
 
 
 /* Error handler called when `obstack_chunk_alloc' failed to allocate
